tests/61push/01message-pushed.pl

```diff
@@ -112,6 +123,40 @@
             die "Unexpected message body";
 
          pass "Alice was pushed";  # Alice has gone down the stairs
+
+         Future->needs_all(
+            await_http_request( "/alice_push", sub {
+               my ( $request ) = @_;
+               my $body = $request->body_from_json;
+
+               return unless $body->{notification}{counts};
+               return 1;
+            })->then( sub {
+               my ( $request ) = @_;
+
+               $request->respond( HTTP::Response->new( 200, "OK", [], "" ) );
```

Comment:
Does this need to be an actually-empty body? You didn't supply a `Content-Length` or connection close or chunked encoding, so bad things will happen. Probably easiest if you don't need the actually-empty response is to

    $request->respond_json( {} );

to have it fill in the content length at least.